deploy/operator.yaml

```diff
@@ -17,13 +17,21 @@ spec:
       containers:
         - name: anthill
           # Replace this with the built image name
-          image: REPLACE_IMAGE
+          image: quay.io/gluster/anthill:latest
```

Comment:
nit: We dont need to specify "latest" tag.

Reply:
True. Though I like it being explicit since for releases this will be set to a version tag.